cmd/cobra_types.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type PoolIds struct {
+	value *[]uint64
+}
+
+func NewPoolIds(pools string) *PoolIds {
+	poolsSlice := parse(pools)
+
+	return &PoolIds{value: &poolsSlice}
+}
+
+func (p *PoolIds) Set(pools string) error {
+	*p.value = parse(pools)
```

Comment:
Perhaps `Set` could reset `p.value` when `pools == ""` and append otherwise.

Reply:
And you're right, this is also how they did it for int64slices; with keeping a `modified` var and, based on that, you can append or replace. I will do that. Thanks

Reply:
oki done